masonry/src/text/selection.rs

```diff
@@ -686,27 +627,43 @@ impl<Str: Deref<Target = str> + TextStorage> Selectable for Str {
     }
 }
 
-/// A cursor type that implements `EditableTextCursor` for string types
+/// A cursor type with helper methods for working with strings.
```

Comment:
```suggestion
/// A cursor type with helper methods for moving through strings.
```
This helps make it clear to readers that the methods are mutating a cursor position